guider.cpp

```diff
@@ -487,7 +487,7 @@ bool Guider::PaintHelper(wxAutoBufferedPaintDCBase& dc, wxMemoryDC& memDC)
 
                     if (newWidth > 0 && newHeight > 0)
                     {
-                        m_displayedImage->Rescale(newWidth, newHeight, wxIMAGE_QUALITY_HIGH);
+                        m_displayedImage->Rescale(newWidth, newHeight, wxIMAGE_QUALITY_BILINEAR);

```

Comment:
> insignificant impact on image quality

It would be really helpful to see a picture of what the difference looks like in the PR description.  We'd like to take your word for it but a picture would be better!

Reply:
A screenshot before this change
![old](https://github.com/OpenPHDGuiding/phd2/assets/152005569/ad43eb8f-b436-405d-9763-53e5add5630b)

Reply:
A screenshot after the change - demonstrates sharper details in the starfield.
![Uploading new.jpg…]()